atlite/gis.py

```diff
@@ -213,8 +239,8 @@ def regrid(ds, dimx, dimy, **kwargs):
     kwargs.update(dst_shape=dst_shape,
                   src_transform=src_transform,
                   dst_transform=dst_transform)
-    kwargs.setdefault("src_crs", 'longlat')
-    kwargs.setdefault("dst_crs", 'longlat')
+    kwargs.setdefault("src_crs", dict(init='EPSG:4326'))
+    kwargs.setdefault("dst_crs", dict(init='EPSG:4326'))
```

Comment:
In `geopandas>=0.7.0` this should be (I think)
```suggestion
    kwargs.setdefault("src_crs", 'EPSG:4326')
    kwargs.setdefault("dst_crs", 'EPSG:4326')
```